src/lib.rs

```diff
@@ -175,38 +176,45 @@ pub fn next() {
 }
 
 /// use with if expressions, e.g. `if MU.t(..) { .. } else { .. }`
-pub fn t(t: bool, n: usize) -> bool {
+pub fn t(t: bool, n: usize, flag: &AtomicUsize, mask: usize) -> bool {
+    report_coverage(n..(n + 3), flag, mask);
     MU.t(t, n)
 }
 
 /// use with while expressions, e.g. `while Mu.w(..) { .. }`
 ///
 /// this never leads to infinite loops
-pub fn w(t: bool, n: usize) -> bool {
+pub fn w(t: bool, n: usize, flag: &AtomicUsize, mask: usize) -> bool {
+    report_coverage(n..(n + 1), flag, mask);
     MU.w(t, n)
 }
 
 /// use instead of `==`
-pub fn eq<R, T: PartialEq<R>>(x: &T, y: &R, n: usize) -> bool {
+pub fn eq<R, T: PartialEq<R>>(x: &T, y: &R, n: usize, flag: &AtomicUsize, mask: usize) -> bool {
+    report_coverage(n..(n + 3), flag, mask);
     MU.eq(x, y, n)
 }
 
 /// use instead of `!=`
-pub fn ne<R, T: PartialEq<R>>(x: &T, y: &R, n: usize) -> bool {
+pub fn ne<R, T: PartialEq<R>>(x: &T, y: &R, n: usize, flag: &AtomicUsize, mask: usize) -> bool {
+    report_coverage(n..(n + 3), flag, mask);
     MU.ne(x, y, n)
 }
 
 /// use instead of `>` (or, switching operand order `<`)
-pub fn gt<R, T: PartialOrd<R>>(x: &T, y: &R, n: usize) -> bool {
+pub fn gt<R, T: PartialOrd<R>>(x: &T, y: &R, n: usize, flag: &AtomicUsize, mask: usize) -> bool {
+    report_coverage(n..(n + 7), flag, mask);
     MU.gt(x, y, n)
 }
 
 /// use instead of `>=` (or, switching operand order `<=`)
-pub fn ge<R, T: PartialOrd<R>>(x: &T, y: &R, n: usize) -> bool {
+pub fn ge<R, T: PartialOrd<R>>(x: &T, y: &R, n: usize, flag: &AtomicUsize, mask: usize) -> bool {
+    report_coverage(n..(n + 7), flag, mask);
```

Comment:
I think this constant offsets may be to fragile. For example, if at some point the amount of mutations for comparisons changes, we may emit an incorrect coverage. What do you think about pass this offsets by parameter (the callers usually have the list of mutations, no?)?